File: GFG/LinkedList/delbeg.cpp
#include<iostream>
#include"../ds.h"
using namespace std;

void delbeg(LinkedList<int> &ll){
    if(ll.head==NULL || ll.head->next==NULL){
        return;
    }
    Node<int>* temp=ll.head;
    ll.head=ll.head->next;
    delete temp;
}

int main(){
    LinkedList<int> tl;
    tl.insert(5);
    tl.insert(10);
    tl.insert(15);
    tl.insert(20);
    delbeg(tl);
    tl.display();
    return 0;
}

File: GFG/LinkedList/delbeg.py
import ds

def deletebeg(ll):
    if(ll.head is None or ll.head.next is None):
        return
    ll.head=ll.head.next
    
tl=ds.LinkedList()
tl.insert(5)
tl.insert(10)
tl.insert(15)
tl.insert(20)
deletebeg(tl)
tl.display()